app/helpers/repositories_helper.rb

```diff
@@ -271,6 +271,50 @@ def show_settings_save_button?(_repository)
       !@repository.managed?
   end
 
+  def breadcrumbs(project:, path:, rev:)
```

Comment:
This method is named `breadcrumb_items` in all other places. I'd like to have it consistent.
```suggestion
  def breadcrumb_items(project:, path:, rev:)
```